assets/css/home.css

```diff
@@ -0,0 +1,101 @@
+
+p{
+    font-size: 1rem;
+    font-family: europa;
+}
+
+.section-1
+{
+    z-index: -1;
+    width: 100%;
+    height: 80vh;
+    
+}
+
+.navbar
+{
+    background-color: transparent !important;
+}
+
+
+.maintexterase{
+    display: none;
+ }
+
+
+.homeimage
```

Comment:
1. If we move the BG-Image to _header.ejs this would also have to be moved to header.css.
2. position:fixed makes more sense or I am missing the logic?

Reply:
The first question has been answered in the last comment sir. For the second one, basically if i set the position property to be fixed then as i scroll the image will move side by side with scrolling, which is not what we are trying to achieve here. I have made the position of the image to be absolute to set it at the top of the page so that it can come in the navbar part also.

Reply:
Okay I thought we are gonna take the header till the end of page. Great!!